tsconfig.json

```diff
@@ -19,7 +19,8 @@
     ],
     "paths": {
       "@/*": ["./*"]
-    }
+    },
+    "baseUrl": "./"
```

Comment:
This was added to address a Cypress warning. Adding `baseUrl` allows Cypress to use relative URLs instead of full ones.